src/mugs.js

```diff
@@ -1,3 +1,37 @@
+/**
+ * A mug is a question, containing data, bind, and control elements.
+ *
+ * Main Properties
+ *  nodeID
+ *  label
```

Comment:
Might want to add [these](https://github.com/dimagi/Vellum/blob/1691e9bba92aebce08e8a28803bff82657003ce0/src/javaRosa.js#L348-L351) too since `mediaItext` is listed below. Technically they're part of the javaRosa plugin, but that plugin is so tightly coupled with the rest of vellum that I think it makes sense to include them here.